src/main/Organization/OrganizationController.java

```diff
@@ -247,9 +375,13 @@ public Handler inviteUsers(SecurityUtils securityUtils, EmailUtil emailUtil) {
       JSONObject req = new JSONObject(ctx.body());
       JSONArray people = req.getJSONArray("data");
 
-//    do a database query to get the request name and org
-      String sender = req.getString("senderName");
-      String org = req.getString("organization");
+      //      String sender = req.getString("senderName");
+      //      String org = req.getString("organization");
+
+      String sender = ctx.sessionAttribute("firstName");
```

Comment:
we can probably make this String sender a one liner. I would also remove the comments above